content/docs/command-reference/exp/show.md

```diff
@@ -258,3 +272,5 @@ sorting only applies to experiment groups (sharing a parent commit).
 
 📖 See [Metrics, Parameters, and Plots](/doc/start/metrics-parameters-plots) for
 an introduction to parameters, metrics, plots.
+
+[regex]: https://en.wikipedia.org/wiki/Regular_expression
```

Comment:
I think reading this page makes people more confused than searching for "regexp." Could you replace it with a simpler page, like https://towardsdatascience.com/an-introduction-to-regular-expressions-5dd762afc5e4 ?

Reply:
Replaced with https://regexone.com/